### ISSUE TITLE
Meet WCAG 2.0 AA(AAA?) for contrast

### ISSUE THREAD

**Issue by Lartza** [nextcloud/server#6727]
Currently there are still a lot of colors that NC uses that do not meet the guidelines for WCAG AA. I feel like this is something that probably is a goal for NC and if not, should be. There have been small commits to fix issues like this in the past but it would be nice to see most of the issues fixed once and for all and then keeping the level for new commits and features.

This came up as a discussion in IRC relating to the Admin page notes about stuff like "Execute one task with each page loaded" but in addition to those there seems to be a lot of other places that use poor contrast. I feel like even I could use the contrast increase for most of the things and I don't have an issue with my eyesight.

So a quick list of the things I found for now that could use some improvement, there's probably more unfortunately:

- settings-hint elements found almost everywhere
- cpuFooterInfo and memFooterInfo under Monitoring
- table column titles in various places like Security, Activity and Logging
- The little "(Stream only)" <em\> under Activity
- "Select tag..." under Workflow > Collaborative tags
- file extension, small filesize and old file text in Files
- The space usage text and bar in bottom left
- Under a file, in Comments your username over the new comment field is very light
- Possibly the box and placeholder too
- Box and placeholder under file's sharing tab ("Name, federated cloud" etc)

@nextcloud/designers @nextcloud/accessibility

**Comment by Lartza** [comment]
Seems like I can't mention groups as I kind of suspected. Also unsure if an edit would have notified anyone so here goes @jancborchardt

**Comment by jancborchardt** [comment]
@Lartza thanks a lot for that review and issue report! :) Do you want to open a pull request for one of the things to start with, for example the settings-hint elements? Best post a before & after screenshot too so we can check how it compares.

**Comment by jancborchardt** [comment]
@Lartza thanks a lot for that review and issue report! :) Do you want to open a pull request for one of the things to start with, for example the settings-hint elements? Best post a before & after screenshot too so we can check how it compares.

Also cc @nextcloud/designers @nextcloud/accessibility
